app/controllers/publish/courses/application_status_controller.rb

```diff
@@ -10,9 +10,14 @@ def new
       end
 
       def update
-        course.update(application_status: 'open')
+        if course.application_status_closed?
+          course.update(application_status: 'open')
```

Comment:
``suggestion
course.application_status_closed!
```

i think we get a few of these methods for free? Same thing for below

Reply:
I thought this might be better: ``` def update
        new_status = course.application_status_closed? ? 'open' : 'closed'

        course.update(application_status: new_status)
        flash[:success] = t("course.application_status.#{new_status}")
        redirect_to publish_provider_recruitment_cycle_course_path
      end```